backend/package.json

```diff
@@ -14,26 +15,49 @@
     "crypto": "0.0.3",
     "express": "4.15.2",
     "express-session": "1.15.3",
+    "mkdirp": "^0.5.1",
```

Comment:
I need to add a `.npmrc` config to the `backend` directory. Maybe this will help getting fix versions.